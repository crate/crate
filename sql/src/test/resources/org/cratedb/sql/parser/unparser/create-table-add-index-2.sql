create table t1 (c1 int, c2 double, index indx1 (c1 ASC) AS IN_MEMORY, c3 int)