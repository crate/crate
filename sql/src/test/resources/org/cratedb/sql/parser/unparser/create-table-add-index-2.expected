CREATE TABLE t1(c1 INTEGER, c2 DOUBLE, INDEX indx1 (c1) AS IN_MEMORY, c3 INTEGER)